boards/qemu_rv32_virt/src/main.rs

```diff
@@ -40,7 +40,7 @@ const FAULT_RESPONSE: kernel::process::PanicFaultPolicy = kernel::process::Panic
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
+pub static mut STACK_MEMORY: [u8; 0x8000] = [0; 0x8000];
```

Comment:
Growing the stack this large was necessary when integrating this with the TAP network driver in `tock-ethernet`, which isn't the most efficient right now (uses an in-kernel ring-buffer of packets). However, given this platform is intended as a test-vehicle for Ethernet development and purely virtual, we can afford it easily.